lib/screen/announcement/model/announcement_model.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:phone_comparison_app/service/database/table/announcement_table.dart';
+
+class Announcement {
+  final int? announcmentId;
+  final String title;
+  final String description;
+  final String category;
+
+  final DateTime? created;
```

Comment:
@irfanSilkron explain kenapa value2 ni mungkin boleh jadi null

Reply:
Make it null sebab when edit the announcement, tanak pass & edit these value (created, updated , isDeleted) . Dah fix it and make it required.